core/src/com/mygdx/crazybomber/Model/bomb/Bomb.java

```diff
@@ -0,0 +1,41 @@
+package com.mygdx.crazybomber.model.bomb;
+
+public class Bomb {
```

Comment:
Might need a field of the Player the bomb is assigned to or the stack that the player uses so you can add it back in.

Reply:
added Stack<Bomb> as argument into the Bomb constructor